server/lease/lessor.go

```diff
@@ -303,6 +303,7 @@ func (le *lessor) Grant(id LeaseID, ttl int64) (*Lease, error) {
 
 	leaseTotalTTLs.Observe(float64(l.ttl))
 	leaseGranted.Inc()
+	leaseTotal.Inc()
```

Comment:
should leaseTotal Inc when etcdserver restart ? the lease may recover from backend in [initAndRecover](https://github.com/etcd-io/etcd/blob/main/server/lease/lessor.go#L790)

Reply:
Thanks for your mentioning @gojoy ,I already updated the commit .
When it recover from backend, the metric will reset the value through leaseTotal.Set(float64(len(le.leaseMap))),
I used to considered doing leaseTotal.Inc inside the recover for-loop, but that would likely have a significant impact.

Reply:
LGTM